src/components/Avatar/Avatar.js

```diff
@@ -0,0 +1,110 @@
+import React, { Component } from 'react';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+import InitialsPropType from '../../prop-types/initials';
+import ListPropType from '../../prop-types/list';
```

Comment:
didn't know about ListPropType, good one

Reply:
ListPropType is something I created!  It's a wrapper of `oneOf` that allows us to access the list of options that we pass to it... we can then display the options in the props table :)